.storybook/common.ts

```diff
@@ -117,10 +116,40 @@ export const saytProductsResponseEvent = {
   }
 };
 
-export function getSaytProductsResponseEvent(): CustomEvent<SaytProductsResponsePayload> {
+export function getSaytProductsResponseEvent(): CustomEvent<any> {
```

Comment:
Ah, that's what you meant by `any`.  You should scope your `any` as small as possible, so you should keep this as-is and cast the object given to `originalResponse` as `any`.